### PR TITLE
Add 0.65 release notes

### PR DESCRIPTION
**Pull request by lewri**
Add release notes to changelog, initial drafting. No date input yet.

Feel free to cherrypick/add/adapt as seem fit.

### CODE REVIEW

**Comment by TheCycoONE** [comment]
GitHub changed the encoding to utf-8, but I'm ok with that.

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+Version 0.65 - released xxxx 2021
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* CorsixTH is now available as a Flatpak! (thanks to @rmnvgr !)
+* On MacOS DMG files are now supported as disk images for Theme Hospital data
+* Hospital policies will now carry over to your next level in the campaign
+* You can now remove destroyed rooms for a fee (config option)
+* The information dialogs that appear when a room is first built can be
+  disabled in the configuration file
+* Fire extinguishers in rooms now have a chance to save a smoking machine
```
Comment (Alberth289346) [inline changelog.txt:11]: + `from exploding` ?

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+* More adviser messages added
```
Comment (Alberth289346) [inline changelog.txt:12]: You can't have less messages added, can you? :) 

More general could be 'Extended advice topics of the advisor', or 'Advisor gives more advice'.
Nicer but more work is to list what the new advice is about.

File: changelog.txt (hunk 1 above)
Comment (lewri) [reply changelog.txt:11]: It is now :)

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+* New trophy added
```
Comment (Alberth289346) [inline changelog.txt:13]: 'New' can be interpreted as 'one more of the original game' and as 'a completely new one', you may want to distinguish between those.

Likely previous entry, listing the one new trophy would be nicer.

(and in a sense more useful for history if we want to find back when "foo trophy" was introduced)

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+* Greek language added! (thanks to dimitrios-iliadis!)
+* Updates to Brazilian-Portuguese language
+* Updates to English language
+* Fixes to Chinese unicode encoding
+
+# Bug Fixes
+* Issues scheduling emergencies have been resolved
+* Games will no longer crash on load if the jukebox was open
+* Games will no longer crash because of an empty music folder
+* Fixed a chance of Doctors getting stuck in a queue after editing a room
```
Comment (Alberth289346) [inline changelog.txt:40]: `Fixed a possibility of Doctors getting stuck in a queue after editing a room` ?

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,55 @@
...
+* Fixed an issue where the game could continue scrolling on load
```
Comment (Alberth289346) [inline changelog.txt:41]: `A game now stops scrolling after loading` ?

**Comment by Alberth289346** [comment]
hmm, too late :(

**Comment by lewri** [comment]
Not fully lost, there will be tweaks before release I imagine =]

**Comment by TheCycoONE** [comment]
Not too late @Alberth289346. Too late is final release. I even made more changes between merging this and the release page that need incorporated.

**Pull request merged**